### PR TITLE
[FolderWatcher] Formatting and write out of abbreviations in Docs

### PR DESCRIPTION
**Pull request by felixschndr**
- `s` --> `in seconds`
- `h` --> `in hours`
- formatting of tables, things and rules
- Unifiy usage of `'` and `"`

### CODE REVIEW

**Comment by goopilot** [comment]
@felixschndr could you hold on with this pull request as there is another one that is also fixing formatting in the readme. https://github.com/openhab/openhab-addons/pull/14669
Pls have a look at it.

**Review by goopilot — CHANGES_REQUESTED** [review]
@felixschndr could you hold on with this pull request as there is another one that is also fixing formatting in the readme. https://github.com/openhab/openhab-addons/pull/14669
Pls have a look at it.

**Comment by felixschndr** [comment]
Okay, will rebase once the PR is merged and then updates this PR

**Comment by felixschndr** [comment]
@goopilot I rebased the PR on your merge into main. Just some small cosmetics to the docs.

**Comment by felixschndr** [comment]
Any news @goopilot ?

File: bundles/org.openhab.binding.folderwatcher/README.md [hunk 1]
```diff
@@ -67,39 +67,33 @@ folderwatcher:s3bucket:myS3bucket       [ s3BucketName="mypublic-bucket", pollIn
 
 FTP example:
```
[the remark this answers is not in the document]
Comment (felixschndr) [reply bundles/org.openhab.binding.folderwatcher/README.md]: Now good?

**Review by goopilot — APPROVED** [review]

**Review by jlaur — APPROVED** [review]
Thanks!

**Pull request merged**